docs/community/tutorials-contest/2022/avax-subnet-customization/README.md

```diff
@@ -0,0 +1,975 @@
+# Avalanche - Creating a Genesis File For Your Subnet
+
+## Introduction
+
+Hi there! In this tutorial, we will be learning how to create a custom genesis file for your Subnet.
+
+We will start by deploying a very simple Subnet using the Subnet wizard. Then we will learn about the genesis file and how to customize it for our needs.
```

Comment:
Link to the `avalanche-cli` like so

"... using [the Subnet Wizard](https://github.com/ava-labs/avalanche-cli) ..."